docs/modules/ROOT/pages/explanations/commodore-components/kustomizations.adoc

```diff
@@ -0,0 +1,116 @@
+= Using kustomizations in Commodore components
+
+[IMPORTANT]
+====
+To use kustomizations, a `kustomize` binary must be present on the system.
+====
+
+It's considered best practice to expose kustomization inputs to the user by introducing a component parameter `kustomize_input`
+
+This minimizes the need for duplicated parameters between the kustomization configuration and the component default parameters.
+
+Additionally, it's considered best practice to expose the kustomization URL in parameter `kustomization_url` and the kustomization version (if applicable) in parameter `manifests_version`.
+
+Finally, it's considered best practice to expose the container images used by the kustomization as documented in the xref:explanations/commodore-components/container-images.adoc[container image versions] best practice documentation.
+
+We recommend that the component renders an overlay which refers to the kustomization in Jsonnet, and calls Kustomize on the resulting YAML file.
+
+
+== Example
+
+In this example, we present a fictional component `slo` which renders a kustomization for `fancy-sli-controller`.
+
+
+In `class/defaults.yml`, we provide the parameters to use when rendering the kustomization, as discussed above.
+
+.class/defaults.yml
+[source,yaml]
+----
+parameters:
+  slo:
+    namespace: syn-slo
+    kustomization_url: https://syn.example.com/fancy-sli-controller//config/default
```

Comment:
```suggestion
    kustomization_url: https://syn.example.com/fancy-sli-controller/config/default
```

Reply:
The `//` is required to separate the repository from the path inside the repository